x-pack/plugins/ml/public/application/trained_models/models_management/test_models/models/inference_info.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC } from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiSpacer, EuiCallOut, EuiText } from '@elastic/eui';
+
+interface Props {
+  taskLabel: string;
+  info: string[];
+}
+
+export const getInferenceInfoComponent = (taskLabel: string, info: string[]) => (
+  <InferenceInfo taskLabel={taskLabel} info={info} />
+);
+
+const InferenceInfo: FC<Props> = ({ taskLabel, info }) => {
+  if (info.length === 0) {
+    return null;
+  }
+  return (
+    <>
+      <EuiCallOut
+        title={i18n.translate('xpack.ml.trainedModels.testModelsFlyout.calloutTitle', {
+          defaultMessage: 'Task: {taskLabel}',
```

Comment:
Seeing the latest changes, IMO it would look even better if we just drop "Task:". WDYT?
```suggestion
          defaultMessage: '{taskLabel}',
```